docs/resources/scim_integration.md

```diff
@@ -25,13 +28,16 @@ resource "snowflake_scim_integration" "aad" {
 
 ### Required
 
-- `name` (String) Specifies the name of the SCIM integration. This name follows the rules for Object Identifiers. The name should be unique among security integrations in your account.
-- `provisioner_role` (String) Specify the SCIM role in Snowflake that owns any users and roles that are imported from the identity provider into Snowflake using SCIM.
+- `enabled` (Boolean) Specify whether the security integration is enabled.
+- `name` (String) String that specifies the identifier (i.e. name) for the integration; must be unique in your account.
+- `run_as_role` (String) Specify the SCIM role in Snowflake that owns any users and roles that are imported from the identity provider into Snowflake using SCIM.
 - `scim_client` (String) Specifies the client type for the scim integration
 
 ### Optional
 
-- `network_policy` (String) Specifies an existing network policy active for your account. The network policy restricts the list of user IP addresses when exchanging an authorization code for an access or refresh token and when using a refresh token to obtain a new access token. If this parameter is not set, the network policy for the account (if any) is used instead.
+- `comment` (String) Specifies a comment for the integration.
+- `network_policy` (String) Specifies an existing network policy that controls SCIM network traffic.
```

Comment:
let's keep track of the newly added fields (and other changes), so that we can communicate them more easily (with release notes or maybe we'll come up with some other way).

For now, please create a new MD file in `v1-preparations`, e.g. changes-before-v1 (or a better name), please put an explanation sentence (that this is a changelog in resources and datasources as part of the https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md#preparing-essential-ga-objects-for-the-provider-v1) and v0.91.0-v0.92.0 changes (for now with your resource section only; @sfc-gh-jcieslak will add database and I will add warehouse).

Reply:
Done.